src/main/java/org/candlepin/resource/OwnerResource.java

```diff
@@ -776,12 +776,15 @@ protected void populateEntity(Pool entity, PoolDTO dto) {
 
     @Override
     @Wrapped(element = "owners")
-    public CandlepinQuery<OwnerDTO> listOwners(String keyFilter) {
-        CandlepinQuery<Owner> query = keyFilter != null ?
+    public Iterable<OwnerDTO> listOwners(String keyFilter) {
+        List<Owner> query = keyFilter != null ?
             this.ownerCurator.getByKeys(Arrays.asList(keyFilter)) :
             this.ownerCurator.listAll();
 
-        return this.translator.translateQuery(query, OwnerDTO.class);
+        return query.stream().map(this.translator.getStreamMapper(
```

Comment:
Let's drop the `.map` call down to its own line so the inner `getStreamMapper` call doesn't need to be broken up across three lines.

Additionally, we can return a Stream rather than an Iterable to avoid needing to convert the stream back into a list.

```
return query.stream()
  .map(this.translator.getStreamMapper(Owner.class, OwnerDTO.class));
```